### PR TITLE
cdc,api: restructure API files

### PR DESCRIPTION
**Pull request by overvenus**
### What problem does this PR solve?

Issue Number: ref #2411

### What is changed and how it works?

* Move api to docs/swagger
* Move cdc/capture http files to cdc/api
* Move cdc/ http files to cdc/api

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

 - Has exported function/method change
 - Has exported variable/fields change

### Release note 

```release-note
None
```

### CODE REVIEW

File: cdc/api/owner.go [hunk 1]
```diff
@@ -72,41 +74,56 @@ func (c ChangefeedResp) MarshalJSON() ([]byte, error) {
 	})
 }
 
+// ownerAPI provides capture APIs.
```
Comment (asddongmen) [inline cdc/api/owner.go:77]:
```suggestion
// ownerAPI provides owner APIs.
```

File: pkg/cmd/cli/cli_changefeed_helper.go [hunk 2]
```diff
@@ -178,9 +178,9 @@ func sendOwnerAdminChangeQuery(ctx context.Context, etcdClient *etcd.CDCEtcdClie
 	}
 
 	resp, err := httpClient.PostForm(url, map[string][]string{
-		cdc.APIOpVarAdminJob:           {fmt.Sprint(int(job.Type))},
-		cdc.APIOpVarChangefeedID:       {job.CfID},
-		cdc.APIOpForceRemoveChangefeed: {forceRemoveOpt},
+		api.APIOpVarAdminJob:           {fmt.Sprint(int(job.Type))},
```
Comment (asddongmen) [inline pkg/cmd/cli/cli_changefeed_helper.go:181]:
```suggestion
		api.OpVarAdminJob:           {fmt.Sprint(int(job.Type))},
```
May be we can remove the prefix `API` from these exported constant.

**Comment by asddongmen** [comment]
rest LGTM

**Review by asddongmen — APPROVED** [review]

**Review by hi-rustin — APPROVED** [review]
great!

**Pull request merged**